<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>s21::painter</class>
 <widget class="QWidget" name="s21::painter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>875</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>painter</string>
  </property>
  <widget class="QGraphicsView" name="painter_obj">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>661</width>
     <height>661</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="predict_button">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>100</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_nah">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>220</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>очистить нах</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>315</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
